### PR TITLE
[image_picker][android] Non-bitmap images now return path instead of null

### PR DESCRIPTION
**Pull request by andreisas06**
## Description
This PR now returns the path of unsupported files(non-bitmap images) instead of null. Now the user can decide himself what to do with the path of the unsupported image. Example file showcases how to display an error placeholder whenever an unsupported image type is selected.

## Issues

Fixes flutter/flutter#88461

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by gmackall** [comment]
Thanks for the contribution! It looks like the tests you added currently are failing to run, if you can get that sorted then we can get this fully reviewed. Also ccing @tarrinneal for image picker related PR visibility.

File: packages/image_picker/image_picker/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.8.7+2
 
+* Example file showcases how to display an error placeholder whenever an unsupported image type is selected.
```
Comment (stuartmorgan) [inline packages/image_picker/image_picker/CHANGELOG.md:3]: We generally describe the change with a verb. E.g., "Adds handling of unsupported image types to the example."

**Review by stuartmorgan — APPROVED** [review]
LGTM with README nits! @gmackall or @tarrinneal could you do the secondary review?

**Review by tarrinneal — APPROVED** [review]

File: packages/image_picker/image_picker_android/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.8.6+7
 
+* Non-bitmap images return path instead of null.
```
[the remark this answers is not in the document]
Comment (andreisas06) [reply packages/image_picker/image_picker_android/CHANGELOG.md]: Updated the PR with the changes, thank you for the review 🙏🏻.

**Pull request merged**